src/Css/Style.php

```diff
@@ -999,6 +999,10 @@ protected function single_length_in_pt(string $l, float $ref_size = 0, ?float $f
 
         $font_size = $font_size ?? $this->__get("font_size");
 
+        if (strncmp($l, 'calc(', 5) == 0) {
+            return $this->_compute_calc($this->calcParse(substr($l, 5, -1)), $ref_size, $font_size);
```

Comment:
`single_length_in_pt` was left in it's current location, but if the evaluation fails at any point null is returned. Failure can be caused by:
- no values to parse
- lack of operator for two non-operator values (e.g. `calc(1 2 3)`).
- failed length calculation

Reply:
There is a lack of validation in the parsing method, but since the logic was written for this specific use case (not for general use) I think I'm OK with that. Errors in the `calc` structure will be caught when attempting to evaluate. We shouldn't see much impact on rendering unless the CSS is excessively `calc`'d.

Reply:
Should be fine, yes, since both methods are private and used in conjunction.